src/main/java/eu/dissco/core/digitalmediaprocessor/service/ProcessingService.java

```diff
@@ -111,59 +130,104 @@ private Set<DigitalMediaObjectEvent> removeDuplicatesInBatch(
     return uniqueSet;
   }
 
-  private void republishEvent(DigitalMediaObjectEvent event) {
+  private void republishEvent(DigitalMediaEvent event) {
     try {
-      kafkaService.republishDigitalMediaObject(event);
+      kafkaService.republishDigitalMedia(event);
     } catch (JsonProcessingException e) {
       log.error("Fatal exception, unable to republish message due to invalid json", e);
     }
   }
 
-  private ProcessResult processDigitalMedia(Set<DigitalMediaObjectEvent> events) {
+  private ProcessResult processDigitalMedia(Set<DigitalMediaEvent> events) {
     var currentMediaObjects = getCurrentDigitalMedia(
-        events.stream().map(DigitalMediaObjectEvent::digitalMediaObjectWrapper).toList());
-    var equalMediaObjects = new ArrayList<DigitalMediaObjectRecord>();
+        events.stream().map(DigitalMediaEvent::digitalMediaWrapper).toList());
+    var equalMediaObjects = new ArrayList<DigitalMediaRecord>();
     var changedMediaObjects = new ArrayList<UpdatedDigitalMediaTuple>();
-    var newMediaObjects = new ArrayList<DigitalMediaObjectEvent>();
+    var newMediaObjects = new ArrayList<DigitalMediaEvent>();
 
     for (var mediaObject : events) {
```

Comment:
change to `for (var digitalMedia : events)`

Reply:
:+1: